Dockerfile

```diff
@@ -14,24 +14,9 @@ RUN apk add --update --no-cache \
 
 ENV RESTIC_BINARY=/usr/local/bin/restic
 
-COPY wrestic/wrestic.sh /app/wrestic
 COPY --from=restic/restic:0.12.1 /usr/bin/restic $RESTIC_BINARY
 COPY k8up /usr/local/bin/
 
-RUN chmod a+x /usr/local/bin/k8up /usr/local/bin/restic /app/wrestic
+RUN chmod a+x /usr/local/bin/k8up /usr/local/bin/restic
 
 USER 65532
-
-## wrestic compatibility shim
-FROM k8up as wrestic
-
-WORKDIR /app
-ENTRYPOINT ["./wrestic"]
-
-## wrestic root compatibility shim
-FROM wrestic as wrestic-root
-
-USER 0
```

Comment:
This PR basically removes the possibility to have the image run as root, which is required in some scenarios: https://github.com/vshn/wrestic/pull/19

We should consider at least allowing to set the security context as an alternative, see https://github.com/vshn/k8up/issues/263

Reply:
I guess https://github.com/vshn/k8up/issues/480 and #263 are the same ^_^

TBH, I don't like to bring back the 'root' Docker image, as it's a work-around for not being able to influence the security context. But it is definitely the "quicker" way.

I suggest to give an implementation of #263 / #480 a time-boxed shot. What do you think @ccremer @tobru ?

Reply:
Yeah, #480 tries to solve the same problem, and yeah, I don't like the root image variants either, it was a workaround.
I prefer #263 though, as it gives more flexibility for the end users. Since we are aiming towards a breaking change, I see this as the only real and intended usage for backing up with root permissions.

Reply:
#480 would be an improvement and guarantee feature parity, as (previously) the restic image could also only be configured on operator level, if I'm not mistaken. Also I believe #480 would be the smaller effort to implement. Plus, once it is implemented, it also helps #263.

Reply:
I'm not sure, it looks about to be the same effort. What speaks against adding a field for the CRDs and take that security context "verbatim" to the generated jobs?

Reply:
I think you make a good argument there, Chris. I haven't thought about that solution, which might make this easier than I feared.

Reply:
I'd say let's go for #263, I like that approach much more :+1:

Reply:
With #499 merged (and therefore #263 resolved) declare this discussion as resolved.